scheduler/src/cook/kubernetes/api.clj

```diff
@@ -779,13 +790,20 @@
                    "does not match pod name argument (" pod-name ")"))
       (log/info "Launching pod with name" pod-name "in namespace" namespace ":" (Yaml/dump pod))
       (try
-        (create-namespaced-pod api namespace pod)
+        ; The purpose of this metric is to help diagnose if the launch workload changes or if k8s response to the
+        ; workload changes (e.g., latency spike)
+        (timers/time! (metrics/timer "launch-pod" compute-cluster-name)
+          (create-namespaced-pod api namespace pod))
         true
         (catch ApiException e
           (let [code (.getCode e)
                 bad-pod-spec? (contains? #{400 404 422} code)]
             (log/info e "Error submitting pod with name" pod-name "in namespace" namespace
                       ", code:" code ", response body:" (.getResponseBody e))
```

Comment:
Now that we have `compute-cluster-name` in scope, would be nice to add it to this log.

Reply:
Ok.